.travis.yml

```diff
@@ -39,3 +39,7 @@ jobs:
         on:
           tags: true
           repo: resonai/ybt
+
+branches:
+  only:
+  - master
```

Comment:
what's this?

Reply:
We don't want to run Travis on every push to every branch, Just on PRs and on pushes to master.
That's how it works in our repository and also Travis began taking money for the runs.

Reply:
cool - can we do this as a separate PR? doesn't seem related to this one :-)